packages/vc-verification/src/errors.ts

```diff
@@ -73,3 +73,16 @@ export class NoIssuers extends Error {
     super(`Issuers for credential ${credential} were not set`);
   }
 }
+
+export const ERRORS = {
+  NoCredential: 'No authorative credential found for issuer',
```

Comment:
Why not get these messages from errors.ts?

Reply:
Like, individual errors? I think it is easy to get all the errors throw single ERROR JSON obj and this becomes easy to export as well (no need to import individually).

Reply:
I would import them since they are same errors, but it is up to you